lib/ra10ke/resolver.rb

```diff
@@ -0,0 +1,56 @@
+require 'puppetfile-resolver'
+require 'puppetfile-resolver/puppetfile/parser/r10k_eval'
+
+module Ra10ke
+  module Resolver
+
+    def define_task_resolver(*args)
+      desc 'Run the puppetfile Resolver'
+      task :resolver do
+        resolver = Ra10ke::Resolver::Instance.new(get_puppetfile.puppetfile_path)
+        result = resolver.resolve
+        # Output resolution validation errors
+        result.validation_errors.each { |err| puts "Resolution Validation Error: #{err}\n"}
+      end
+    end
+
+    class Instance
+      attr_reader :puppetfile
+
+      def initialize(puppetfile_path = File.expand_path(Dir.pwd))
```

Comment:
Isn't this a directory, not a file? Cant read a dir

Reply:
Good catch. why yes that would fail.